tests.py

```diff
@@ -48,6 +49,17 @@ def test_clean_input(self):
                 "/", data={'password': 'foo', 'ttl': 'hour'}, method='POST'):
             self.assertEqual((3600, 'foo'), snappass.clean_input())
 
+    def test_password_before_expiration(self):
+        password = 'fidelio'
+        key = snappass.set_password(password, 1)
+        self.assertEqual(password, snappass.get_password(key))
+
+    def test_password_after_expiration(self):
+        password = 'open sesame'
+        key = snappass.set_password(password, 1)
+        time.sleep(1.5)
+        self.assertEqual(None, snappass.get_password(key))
```

Comment:
I actually thought of using it, but it was only [introduced in python 2.7](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsNone), and the tests still run on 2.6.